src/controllers/JobRoleController.ts

```diff
@@ -0,0 +1,12 @@
+import express from "express";
+import { getJobRoles } from "../services/JobRoleService";
+
+export const getAllJobRoles = async (req: express.Request, res: express.Response): Promise<void> => {
+    try{
+        res.render('jobRoles', { jobRoles: await getJobRoles() });
+    } catch (error){
+        res.locals.errormessage = error.message;
+        res.render('jobRoles');
```

Comment:
should this not say `res.render('jobRoles.html');` ? and above in the try{} block